c_src/build_deps.sh

```diff
@@ -30,6 +30,18 @@ MAKE=${MAKE:-make}
 
 # Changed "make" to $MAKE
 
+get_dep_leveldb () {
+    if [ ! -d leveldb ]; then
+        git clone git://github.com/basho/leveldb
+        (cd leveldb && git checkout $LEVELDB_VSN)
+        if [ "$BASHO_EE" = "1" ]; then
+            (cd leveldb && git submodule update --init)
+        fi
+    else
+        (cd leveldb && dl=$(git diff $LEVELDB_VSN |wc -l) && [ $dl != 0 ] && >&2 echo "\033[0;31m WARN - local leveldb is out of sync with remote $LEVELDB_VSN\033[0m") || :
```

Comment:
The ANSI color codes work in bash and dash. By doesn't seem to handle, if you mean it doesn't show color but otherwise functions correctly, I would not consider this to be an issue.